lib/utils/addSemicolonForEditInfo.mjs

```diff
@@ -0,0 +1,30 @@
+/** @import { Node as PostcssNode } from 'postcss' */
+
+/**
+ * Adjust the text in EditInfo to include a semicolon when needed.
+ *
+ * @param {PostcssNode} node
+ * @param {{range: [number, number], text: string}} fixData
+ * @returns {{range: [number, number], text: string}}
+ */
+export default function addSemicolonForEditInfo(node, fixData) {
```

Comment:
[suggestion] Minor. We probably could check `parent` first to avoid repeated its evaluation within each `if` block:

```suggestion
export default function addSemicolonForEditInfo(node, fixData) {
	const { parent } = node;

	if (!parent) return fixData;

```

Reply:
I don't think that the parent can be undefined unless the root node is being passed, is that what we want to check here?

Reply:
Yes!